scripts/get_digest.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+IMAGE=$1
+TAG=$2
+
+CURL=`which curl`
```

Comment:
why using `which` here?

Reply:
For some reason I was having trouble using the commands directly in the script, something like:
```
curl: command not found

sed: command not found

jq: command not found
```
I found it safer to do it that way.